installing/installing_with_agent_based_installer/installing-using-iscsi.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-using-iscsi"]
+= Preparing installation assets for iSCSI booting
+include::_attributes/common-attributes.adoc[]
+:context: installing-using-iscsi
+
+toc::[]
+
+You can boot an {product-title} cluster through iSCSI using an ISO image generated by the Agent-based Installer.
+The following procedures describe how to prepare the necessary installation resources to boot off of an iSCSI target.
+
+The assets you create in these procedures will deploy a single-node {product-title} installation.
+You can use these procedures as a basis and modify configurations according to your requirements.
+
+[id="iscsi-boot-requirements_{context}"]
+== Requirements for iSCSI booting
+
+The following configurations are necessary to enable iSCSI booting using the Agent-based Installer:
+
+* Dynamic Host Configuration Protocol (DHCP) must be configured.
+Static networking is not supported.
+* You must create an additional network for iSCSI that is separate from the machine network of the cluster.
+The machine network is rebooted during cluster installation and cannot be used for the iSCSI session.
+* If the host on which you are booting the agent ISO image also has an installed disk, it might be necessary to specify the iSCSI disk name in the `rootDeviceHints` parameter to ensure that it is chosen as the boot disk for the final {op-system-first} image.
+You can also use a diskless environment for iSCSI booting, in which case you do not need to set the `rootDeviceHints` parameter.
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#agent-install-networking-DHCP_preparing-to-install-with-agent-based-installer[DHCP]
+* xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc#root-device-hints_preparing-to-install-with-agent-based-installer[About root device hints]
+
+
+// I took the following prerequisites from the main Agent install procedure, but let me know if any of these do not apply/ do not make sense for iSCSI booting
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/overview/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* If you use a firewall or proxy, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
+
+// Downloading the Agent-based Installer
+include::modules/installing-ocp-agent-download.adoc[leveloffset=+1]
+
+// Creating the preferred configuration inputs
```

Comment:
Might be out of scope, but the last note under the callouts dangles: 

![Screenshot From 2025-02-17 17-02-06](https://github.com/user-attachments/assets/13d7a8b0-c692-4f78-a17a-10003f447ce3)

Reply:
If I remember correctly that is intentionally _not_ nested under that callout since it applies to the overall step/source block, not just callout 6